src/views/catalog/wizard/tabs/network/components/list/NetworkInterfaceActions.tsx

```diff
@@ -8,23 +8,27 @@ import TabModal from '@kubevirt-utils/components/TabModal/TabModal';
 import KebabToggle from '@kubevirt-utils/components/toggles/KebabToggle';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { NetworkPresentation } from '@kubevirt-utils/resources/vm/utils/network/constants';
+import { vmSignal } from '@kubevirt-utils/store/customizeInstanceType';
 import { ButtonVariant, Dropdown, DropdownItem, DropdownList } from '@patternfly/react-core';
 
 import WizardEditNetworkInterfaceModal from '../modal/WizardEditNetworkInterfaceModal';
 
 type NetworkInterfaceActionsProps = {
   nicName: string;
   nicPresentation: NetworkPresentation;
+  onUpdateVM?: (updateVM: V1VirtualMachine) => Promise<void>;
 };
 
 const NetworkInterfaceActions: FC<NetworkInterfaceActionsProps> = ({
   nicName,
   nicPresentation,
+  onUpdateVM,
 }) => {
   const { t } = useKubevirtTranslation();
-  const { updateVM, vm } = useWizardVMContext();
+  const { updateVM, vm: vmContext } = useWizardVMContext();
   const { createModal } = useModal();
-
+  const vm = vmSignal.value || vmContext;
```

Comment:
Is this a temporary solution or will we always use context as a fallback? If it's not temporary, what's the value in having two store solutions?

Reply:
in the end i would like to remove all use of context and stay only with signals so its temp until a wider refactor